frameworks/hardhat-advanced/test/sample-test.js

```diff
@@ -2,7 +2,7 @@ const { expect } = require("chai");
 const { ethers } = require("hardhat");
 
 describe("Hardhat advanced: Greeter", function () {
-  it("Should return the new greeting once it's changed", async function () {
+  it.skip("Should return the new greeting once it's changed", async function () {
```

Comment:
we need to fix the neonlabs objects [here](https://github.com/neonlabsorg/neon-compatibility/blob/3e642170cdd449057d8d4ac85997c376a3d091d2/frameworks/hardhat-advanced/hardhat.config.js#L37)

Please notice the HDWalletProvider object in the [config](https://github.com/neonlabsorg/neon-compatibility/blob/3e642170cdd449057d8d4ac85997c376a3d091d2/frameworks/hardhat-advanced/hardhat.config.js#L37)